test/unit/worker.js

```diff
@@ -191,6 +344,29 @@ describe('Worker', function () {
               // this is asserting taskHandler called once, but was twice
               assert.ok(taskHandler.calledOnce, 'task was called once')
               assert.ok(doneHandler.calledOnce, 'done was called once')
+              sinon.assert.callCount(monitor.increment, 5)
+              sinon.assert.calledWith(monitor.increment.firstCall, 'ponos', {
+                token0: 'command',
+                token1: 'something.command',
+                token2: 'do.something.command',
+                queue: 'do.something.command'
+              })
+              sinon.assert.calledWith(monitor.increment.secondCall, 'ponos.finish', {
+                result: 'timeout-error',
+                token0: 'command',
+                token1: 'something.command',
+                token2: 'do.something.command',
+                queue: 'do.something.command'
+              })
+              sinon.assert.calledTwice(monitor.timer)
+              sinon.assert.calledWith(monitor.timer, 'ponos.timer',
+                !process.env.WORKER_MONITOR_DISABLED, {
```

Comment:
then this sneaky `process.env` shows up again. I keep thinking it's a value in some object that is defined, but it's an entirely new argument!